src/main/java/net/syscon/elite/security/AuthenticationFacade.java

```diff
@@ -44,14 +44,15 @@ public String getCurrentUsername() {
         return username;
     }
 
-    public boolean isIdentifiedAuthentication() {
+    public String getProxyUserAuthenticationSource() {
```

Comment:
wondering if this should be an enum as values are known?

Reply:
Yeah, good idea.